app/templates/Event/details.html.twig

```diff
@@ -61,7 +61,9 @@
 {% endblock %}
 
 {% block extra_meta %}
-    {{ parent() }}
```

Comment:
It's the other way around. We inherit from layout in this view, so want to pick up any extra_meta in there. I think it's empty at the moment, but we can't assume that it will stay empty in the future.